libs/shared/src/consts/providers/channels/sms.ts

```diff
@@ -162,4 +163,12 @@ export const smsProviders: IProviderConfig[] = [
     docReference: 'https://docs.novu.co/channels/sms/africas-talking',
     logoFileName: { light: 'africas-talking.svg', dark: 'africas-talking.svg' },
   },
+  {
+    id: SmsProviderIdEnum.Kavenegar,
+    displayName: `Kavenegar SMS`,
+    channel: ChannelTypeEnum.SMS,
+    credentials: kavenegarConfig,
+    docReference: 'https://docs.novu.co/channels/sms/kavenegar',
+    logoFileName: { light: 'kavenegar-provider.png', dark: 'kavenegar-novu-sms.png' },
```

Comment:
You need to add the logo image here: https://github.com/novuhq/novu/tree/next/apps/web/public/static/images/providers

Considering the logo we would need to a dark theme version:
https://panel.kavenegar.com/public/v2/images/kavenegar.svg